tests/integration/components/paper-form-test.js

```diff
@@ -110,8 +110,7 @@ module('Integration | Component | paper form', function(hooks) {
       assert.notOk(isInvalidAndTouched);
     });
 
-    await click('input:first-of-type');
-    await blur('input:first-of-type');
+    await triggerEvent('input:first-of-type', 'blur');
```

Comment:
Whu use `triggerEvent` instead of the `blur` helper?

Reply:
Using just `blur` without `click` makes this test fail - I haven’t looked at why - saw some other tests were already using `triggerEvent` - so I did the same.